topdown/json.go

```diff
@@ -176,7 +176,10 @@ func parsePath(path *ast.Term) (ast.Ref, error) {
 	var pathSegments ast.Ref
 	switch p := path.Value.(type) {
 	case ast.String:
-		parts := strings.Split(strings.Trim(string(p), "/"), "/")
+		if p == "" {
+			return ast.Ref{}, nil
+		}
```

Comment:
I wonder if this change affects `json.remove` and `json.filter`, but I haven't played with the code yet.

Reply:
I think there may have been a bug here before: `""` used to refer to the empty key (e.g. `{"": 1}`) before, whereas it now refers to the root path (which seems more appropriate).